client/core/status.go

```diff
@@ -69,9 +69,7 @@ func (c *Core) resolveMatchConflicts(dc *dexConnection, statusConflicts map[orde
 				// but it was revoked in the short period since then.
 				c.log.Errorf("Server did not report a status for match during resolution. %s",
 					statusResolutionID(dc, trade, match))
-				// revokeMatch only returns an error for a missing match ID, and
-				// we already checked in compareServerMatches.
-				_ = trade.revokeMatch(match.id, false)
```

Comment:
This is why the individual resolvers don't have to call `revokeMatch` to get the notifications and return of coins, instead justting set `match.MetaData.Proof.{Self,Server}Revoked`.  Aren't we now doing those actions twice (in the resolver and here)?

Reply:
I'll take another look here, but I'm sure it can be modified to ensure it's not done twice.